packages/ezxenstore/ezxenstore.0.1.2/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "jonathan.ludlam@citrix.com"
+authors: ["xen-api@lists.xensource.com"]
+license: "ISC"
+homepage: "https://github.com/xapi-project/ezxenstore"
+bug-reports: "https://github.com/xapi-project/ezxenstore/issues"
+dev-repo: "https://github.com/xapi-project/ezxenstore.git"
+
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+
+depends: [
+  "jbuilder" {build}
```

Comment:
needs to be a lower bound since `-p` was only added in beta7 i think. Safest to specify beta10 or higher for all these via:

`"jbuilder" {build & >="1.0+beta10"}`